AMP-96489 Add new docs for lookup table apis v3 (#349)
amplitude/amplitude-dev-center: docs/data/sources/lookup-table.md

    - The first column must correspond to the mapping property value and must contain *unique* values. Lookup Tables search for exact matches, and are *case-sensitive*.
    - Columns must be separated by commas.
    - Rows must be separated by line breaks.

>     - If a field value contains commas or quotes, it should be wrapped within double quotation marks. The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by double quotes.

Reviewer: ```suggestion
    - If a field value contains commas or quotes, wrap the contents with double quotation marks (`"`). The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by double quotes.
```
@rctbusk I'm not clear on the last sentence here. It seems like we should tell users to replace double-quotes within a field with single quotes, so they don't interfere with the double-quote wrapper. Is this accurate?

Author: yep! that makes sense.

>     - If a field value contains commas or quotes, wrap the contents with double quotation marks (`"`). The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by double quotes.

Reviewer: ```suggestion
    - If a field value contains commas or quotes, wrap the contents with double quotation marks (`"`). The first double quote signifies the beginning of the column data, and the last double quote marks the end. If a string contains double quotes, replace them with single quotes.
```

1. In Amplitude Data, click **Catalog** and select the **Sources** tab.
2. In the Lookup Tables section, click **CSV**.

> 3. Select column in CSV as the key column.

Reviewer 2: Maybe "Select a column in the CSV to be the key column"?

> 3. Select column in CSV to map property to.

Reviewer 2: Grammar seems a bit off?

Author: Tried fixing a bit
